apps/sasquatch/src/main/res/values/array.xml

```diff
@@ -18,8 +18,8 @@
         <item>Custom</item>
     </string-array>
     <string-array name="appcenter_secret_guids" tools:ignore="MissingTranslation">
```

Comment:
I'd rather just call them appcenter_secrets, type or format does not have to be in the name of the key.

Reply:
Thanks, fixed